recipes/bzip2/all/conanfile.py

```diff
@@ -5,7 +5,6 @@
 
 class Bzip2Conan(ConanFile):
     name = "bzip2"
-    version = "1.0.8"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "http://www.bzip.org"
     license = "bzip2-1.0.8"
```

Comment:
```suggestion
    license = "bzip2"
```

I would prefer not to see some version here.

Reply:
That's part of license name: https://spdx.org/licenses/bzip2-1.0.6.html

Reply:
But at the same time bzip2-1.0.8 license doesn't exist, if I'm right.